build/build-image/Dockerfile

```
@@ -16,6 +16,13 @@
 
 FROM debian:bullseye
 
+ARG USER_ID
+ARG GROUP_ID
```

```suggestion
ARG USER_ID=1337
ARG GROUP_ID=1338
```

https://docs.docker.com/engine/reference/builder/#default-values

Then you don't have to set them separately everywhere 😄